db/schema.sql

```diff
@@ -0,0 +1,30 @@
+DROP DATABASE IF EXISTS hotels;
+
+CREATE DATABASE hotels;
+
+USE hotels;
+
+CREATE TABLE Image (
+  `id` INT AUTO_INCREMENT,
+  `hotel_id` INT NOT NULL,
+  `title` TEXT,
+  `url` TEXT,
+  `author` TEXT,
```

Comment:
I assume the author refers to the ID of the user but looks good.

Reply:
Ideally it will be, yes! I just made it a string for the sake of expediency, so I can work on getting a minimum viable product up and working first.